Code/CSharpOOP3/Flowers.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace CSharpOOP3
+{
+    public class Flowers  //Task_01
+    {
+        public string Color //(setter-only property)
+        {
+            set { Color = value; }
+        }
+
+        public double Price { get { return Price; } }  //(read-only property)
+
+        public string Name//(Task_03)
+        {
+            get { return Name; }
+            private set { Name = value; }
+        }
+        public static string Type { get; set; }
+
+        public static double CalculateTotalPrice(Flowers[] flowersArray)// (Task_5)
```

Comment:
this method is not really connected with class Flowers :(

Reply:
done